src/transformers/models/dinat/modeling_dinat.py

```diff
@@ -21,11 +21,12 @@
 
 import torch
 import torch.utils.checkpoint
-from torch import nn
+from torch import Tensor, nn
```

Comment:
Let's use `torch.Tensor` everywhere for more consistency (it is used in several parts of this file).